iptables/_rule.sls

```diff
@@ -11,6 +11,9 @@ iptables_{{ chain_name }}_{{ rule_name }}:
   {%- endif %}
   - table: {{ rule.get('table', 'filter') }}
   - chain: {{ chain_name }}
+  {%- if rule.family is defined %}
+  - family: {{ rule.family }}
```

Comment:
It's not only about having a firewall but you may also need to setup forward rules and others. Ok, we can keep it as is for now until someone needs it.